charts/sequencer/files/cometbft/config/custom.genesis.json

```diff
@@ -120,12 +120,12 @@
       {{ include "sequencer.address" $value }}
       {{- end }}
     ]
-    {{- if and .Values.global.dev (not .Values.sequencer.upgrades.systemTest.enabled) }},
+    {{- if and .Values.global.dev .Values.genesis.priceFeed }},
```

Comment:
It felt awkward the interdependence here. If you define it in the genesis fine, if you define it in an upgrade fine. It might cause issues to have both but.... don't do that?

The files should only defined this way for testing and local development anyways